test/image/mocks/polar_blank.json

```diff
@@ -33,6 +33,9 @@
         "x": [0, 0.46],
         "y": [0.56, 1]
       },
+      "angularaxis": {
+        "labelalias": { "0": "ZERO<sup>2</sup>" }
```

Comment:
Still I am not 100% sure about this one in polar.
Shouldn't we use the `0°` key instead of `0`?

Reply:
@alexcjohnson Is this case correct?

Reply:
Fixed by [988cde1](https://github.com/plotly/plotly.js/pull/6481/commits/988cde1a8e17d0098fa26b011a9cedee6e99109e).